### PR TITLE
rc4.0: Move upload release report step 

### PR DESCRIPTION
**Pull request by sonalideshpandemsft**
Cherry-pick https://github.com/microsoft/FluidFramework/pull/21887 and https://github.com/microsoft/FluidFramework/pull/21969

### CODE REVIEW

**Review by tylerbutler — APPROVED** [review]

File: tools/pipelines/templates/include-install-build-tools.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      name: PrependPath
+      displayName: Prepend build-tools CLI to path
+      inputs:
+        targetType: 'inline'
+        workingDirectory: ${{ parameters.buildDirectory }}
+        script: |
+          # Prepend the cli bin dir to the path. See
+          # <https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash#prependpath-prepend-a-path-to-the--path-environment-variable>
+          # more information.
+          echo "##vso[task.prependpath]$(Build.SourcesDirectory)/build-tools/packages/build-cli/bin"
```
Comment (alexvy86) [inline tools/pipelines/templates/include-install-build-tools.yml:37]: This bit isn't here in the rc5 PR, is that expected?

Comment (tylerbutler) [reply tools/pipelines/templates/include-install-build-tools.yml:37]: Was this change already in RC5?

Comment (sonalideshpandemsft) [reply tools/pipelines/templates/include-install-build-tools.yml:37]: This bit is present in rc4 but not in rc5.

**Comment by tylerbutler** [comment]
Merging to unblock Loop pipeline work.

**Pull request merged**